README.md

```diff
@@ -1,4 +1,13 @@
 Docker Toolbox
+
+### TL/DR: Docker will be ceasing Support and Development of the Docker Toolbox project in 2020 - going forward please use [Docker Desktop](https://www.docker.com/products/docker-desktop)
```

Comment:
I think the readme should state the current status not the future state. I'd suggest something instead along the lines of:
> This project and repository is now deprecated and is no longer in active development. Please use Docker Desktop instead where possible.

Or a simpler change to use present tense:
> Docker has ceased support and development of the Docker Toolbox project.

Reply:
Took the first suggestion